_components/in-page-navigation/in-page-navigation.md

```diff
@@ -40,24 +40,22 @@ tags:
   - nav
 implementation:
   initProps:
+    - property: "`data-main-content-selector`"
+      description: The element the component pulls headers from.
```

Comment:
Adding `optional` could make it clear to users that this is an enhancement.

```suggestion
      description: The optional element the component pulls headers from.
```

Reply:
I hesitate to add this only because the other properties are also optional. I wouldn't want to mark it on one but not the others. Thoughts?

Reply:
Maybe we should add a heading for the tables stating that they're optional? Quick example

![image](https://github.com/uswds/uswds-site/assets/25211650/b039cb81-48c6-4f54-af3b-b5a9b7f68739)

Reply:
Or is that misleading since they technically have defaults?

Reply:
I added some "If needed" text in to the init props intro   [14cce4e](https://github.com/uswds/uswds-site/pull/2190/commits/14cce4ec0980aced7c5cdcd74404ee6363bd576b) to further suggest that these are not required attributes. I think that having defaults speaks to the fact that they are not required, but happy to be more explicit if you both think it is necessary. 

Related, I'd also be interested in setting up a new issue to standardize the "Using the in-page navigation component" section. It doesn't follow the convention of other pages, so I'd like to clean that up a bit. We could possibly continue this discussion there if needed.

Reply:
> I hesitate to add this only because the other properties are also optional. I wouldn't want to mark it on one but not the others. Thoughts?

That's a good point. Let's leave it out then. 

We can create a new potential enhancement if we find optional settings unclear in this table. **Not a requirement at this time.**